### DISCUSSION TITLE
1.3.17

### DISCUSSION THREAD

**Discussion by benhillis** [microsoft/WSL#10430] [General]
*  Minor networking-related bugfixes.

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WSL/releases/tag/1.3.17'>1.3.17</a>.</em>

**Comment by blakeduffey** [comment]
Is there an expectation of when a release will happen?

1.2.5 was over 4 months ago.

Thanks much.

**Comment by muratyurdakul75** [comment]
When will more CPU usage support than 64 CPUs come? :(
